src/MCPClient/lib/clientScripts/verify_and_restructure_transfer_bag.py

```diff
@@ -55,6 +56,21 @@ def restructureBagForComplianceFileUUIDsAssigned(
         dirPath = os.path.join(unitPath, dir)
         dirDataPath = os.path.join(unitPath, "data", dir)
         if os.path.isdir(dirDataPath):
+            # When moving metadata directory from the bag, check if
+            # there is already a metadata dir at the top level
+            # (it could have been created by automation tools)
+            # (ref. AM issue 1022 )
+            topleveldir_exists = False
+            if dir == "metadata" and os.path.isdir(dirPath):
+                # temporarily rename top level metadata directory
+                job.pyprint(
+                    "{}: temporarily renaming pre-existing directory".format(dir)
```

Comment:
If we've reached this part of the code we know `metadata` is going to be renamed. I'd be tempted to add the path as well so that we can get a complete rendering of exactly where in the structure the folder sits.

Reply:
Good idea. Changed the pyprint message to show the complete path instead.